scripts/analysis.py

```diff
@@ -638,50 +583,171 @@ def final_stockpile(cursor, facility):
     return outstring
 
 
-    return outstring
+def fuel_into_reactors(cursor):
+    """ Finds timeseries of mass of fuel received by reactors
 
- 
-def fuel_usage_timeseries(cursor, fuel_list):
-    """ Calculates total fuel usage over time
+    Parameters
+    ----------
+    cursor: sqlite cursor
+        sqlite cursor
+
+    Returns
+    -------
+    get_timeseries: function
+        Function returns timeseries of mass of fuel into receactors [MTHM]
+    """
+    init_year, init_month, duration, timestep = get_sim_time_duration(cursor)
+    fuel = cursor.execute('SELECT time, sum(quantity) FROM transactions '
+                          'INNER JOIN resources ON '
+                          'resources.resourceid = transactions.resourceid '
+                          'INNER JOIN agententry ON '
+                          'transactions.receiverid = agententry.agentid '
+                          'WHERE spec LIKE "%Reactor%" '
+                          'GROUP BY time').fetchall()
+
+    return get_timeseries(fuel, duration, .001, 'TRUE')
+
+
+def conv_ratio(cursor, in_, out, is_recipe):
+    """ Returns conversion ratio of two commodities
 
     Parameters
     ----------
     cursor: sqlite cursor
         sqlite cursor
-    fuel_list: list
-        list of fuel commodity names (eg. uox, mox) to consider in fuel usage.
+    in_: str
+        name of in recipe / commodity
+    out: str
+        name of out recipe / commodity
+    is_recipe: bool
+        recipe if true, commodity if false
 
     Returns
     -------
-    dict
-        dictionary of different fuels used timeseries
-    keys - fuel
-    values - timeseries list of fuel amount [kg]
+    prints conversion ratio
```

Comment:
I strongly recommend returning a value rather than printing it. Execute print() in __main__ or in the ipython notebook or in an umbrella function, but don't print as the operation of a function. At the very least, both return the important value and print it.